.github/workflows/ci.yml

```diff
@@ -64,12 +64,13 @@ jobs:
       - run: dart format --set-exit-if-changed .
 
   generate:
-    runs-on: ubuntu-20.04
+    runs-on: ubuntu-22.04
```

Comment:
i bumped the ubuntu version because the generator script requires a newer python version. i kept 20.04 for all other jobs because the installer is core20-based